machine-learning/solutions/seismic_salt_detection/trainer/inputs.py

```diff
@@ -44,15 +42,8 @@ def parse_csv(record):
     Returns:
         A dictionary with all column names and values for the record.
     """
-    distribution_defaults = [[0.0] for _ in range(constants.DISTRIBUTION_SIZE)]
-    weather_defaults = [[0.0] for _ in range(constants.WEATHER_SIZE)]
-    distribution_cols = ['distribution' +
-                         str(i) for i in range(constants.DISTRIBUTION_SIZE)]
-    weather_cols = ['weather' + str(i) for i in range(constants.WEATHER_SIZE)]
-    header_def = [[0.0], [''], [0], [0]] + \
-        distribution_defaults + weather_defaults
-    column_names = [TARGET_COLUMN, 'date', 'day',
-                    'hour'] + distribution_cols + weather_cols
+    header_def = [[''], [0], [0]]
```

Comment:
will change name to column_defaults

I named it parse_csv as I'm using same wording TF uses for their function to parse lists of comma separated strings : tf.decode_csv

Reply:
My reading of it is that tf.decode_csv, CSV is a set of records, whereas in parse_csv a CSV is a single record.  Please resolve this without making any edits if this is pedantic/unhelpful rather than pedantic/helpful.